animations/lib/src/basics/05_animated_builder.dart

```diff
@@ -46,16 +46,21 @@ class _AnimatedBuilderDemoState extends State<AnimatedBuilderDemo>
         child: AnimatedBuilder(
           animation: animation,
           builder: (context, child) {
-            return MaterialButton(
-              color: animation.value,
-              child: child,
-              onPressed: () {
-                if (controller.status == AnimationStatus.completed) {
-                  controller.reverse();
-                } else {
-                  controller.forward();
-                }
-              },
+            return Container(
```

Comment:
As I mentioned in the related issue, a full-screen button isn't a UX pattern we'd want to show here. Increasing the height, width, and font size to make the button slightly bigger could work, however.